pages/common/quarto.md

```diff
@@ -0,0 +1,36 @@
+# quarto
+
+> An open-source scientific and technical publishing system built on Pandoc.
+> More information: <https://quarto.org/>.
+
+- Create a Quarto project:
+
+`quarto create-project {{path/to/project}}`
+
+- Create a new website project:
+
+`quarto create-project {{destination_directory}} --type {{website}}`
```

Comment:
```suggestion
`quarto create-project {{path/to/destination_directory}} --type {{website}}`
```

Reply:
Whoops, I thought this was already fixed 😅